### PR TITLE
Remove from/import and update to PHP5.4.

### PR DESCRIPTION
**Pull request by vonglasow**

### CODE REVIEW

File: AdjacencyList.php [hunk 1]
```diff
@@ -34,28 +34,11 @@
...
- * \Hoa\Graph\IGraph\Node
- */
--> import('Graph.I~.Node');
-
-}
-
-namespace Hoa\Graph {
+use Hoa\Graph\Graph;
+use Hoa\Graph\Exception;
+use Hoa\Graph\IGraph;
```
Comment (Hywan) [inline AdjacencyList.php]: We don't need these `use`. We are in the same namespace.

File: Exception.php [hunk 2]
```diff
@@ -34,7 +34,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-namespace Hoa\Graph {
+namespace Hoa\Graph;
```
Comment (Hywan) [inline Exception.php]: Add `use Hoa\Core;` and simplify the `extends` below.

File: Graph.php [hunk 3]
```diff
@@ -275,13 +266,8 @@ public function isLoopAllow ( ) {
...
 
-}
-
-namespace {
-
 /**
  * Flex entity.
  */
-Hoa\Core\Consistency::flexEntity('Hoa\Graph\Graph');
+\Hoa\Core\Consistency::flexEntity('Hoa\Graph\Graph');
```
Comment (Hywan) [inline Graph.php]: Same here.

File: SimpleNode.php [hunk 4]
```diff
@@ -34,18 +34,9 @@
...
-
-/**
- * \Hoa\Graph\IGraph\Node
- */
--> import('Graph.I~.Node');
-
-}
-
-namespace Hoa\Graph {
+use Hoa\Graph\IGraph;
```
Comment (Hywan) [inline SimpleNode.php]: We don't need this `use` neither.

**Comment by Hywan** [comment]
Few notices. Thanks for the great job!

**Comment by vonglasow** [comment]
Notices updated ;)

File: Graph.php [hunk 5]
```diff
@@ -34,18 +34,10 @@
...
-
-/**
- * \Hoa\Graph\Exception
- */
--> import('Graph.Exception');
-
-}
-
-namespace Hoa\Graph {
+use Hoa\Graph\Exception;
```
Comment (Hywan) [inline Graph.php]: `use Hoa\Graph\Exception;` is useless.

File: Graph.php [hunk 6]
```diff
@@ -157,7 +149,7 @@ public static function getInstance ( $type = self::TYPE_ADJACENCYLIST ) {
      * @throw   \Hoa\Graph\Exception
      */
     abstract public function addNode ( \Hoa\Graph\IGraph\Node $node,
```
Comment (Hywan) [inline Graph.php]: We can simplify this type-hint by `IGraph\Node` directly.

**Comment by Hywan** [comment]
Thanks!

**Comment by Hywan** [comment]
http://wiki.hoa-project.net/projects:central:todo is updated

**Pull request merged**